### PR TITLE
Adding verify_app_is_up operation test

### PR DESCRIPTION
**Pull request by raskin-aleksandr**
Adding `verify_app_is_up` operation test
Fix for `verify_app_is_up`

### CODE REVIEW

**Comment by meirwah** [comment]
wait.. some flows this operation, so if you change the input name it will not work anymore...

**Comment by meirwah** [comment]
I would not change the input name (to attempts) , but keep it max_seconds_to_wait, just change the msg output...

File: test/io/cloudslang/base/network/verify_app_is_up.inputs.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - output_message: Application is not up after 0 attempts to ping.
+
+testVerifyAppIsUp_Exception:
+  inputs:
+    - host: 64.233.177.113
+    - port: '80'
+    - attempts: aa
+  description: Tests that verify_app_is_up.sl operation does not contain an error message
+  testFlowPath: io.cloudslang.base.network.verify_app_is_up
+  throwsException: true
```
Comment (oritstone) [inline test/io/cloudslang/base/network/verify_app_is_up.inputs.yaml]: Description of all test cases is wrong. 
Each test case should have a description that describes what you test

File: test/io/cloudslang/base/network/verify_app_is_up.inputs.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - host: 64.233.177.113
+    - port: '80'
+    - attempts: '1'
+  description: Tests that verify_app_is_up.sl operation does not contain an error message
+  testFlowPath: io.cloudslang.base.network.verify_app_is_up
+  result: SUCCESS
+
+testVerifyAppIsUp_Failure:
+  inputs:
+    - host: localhost
```
Comment (oritstone) [inline test/io/cloudslang/base/network/verify_app_is_up.inputs.yaml]: If you want to pass illegal host name, you should put invalid ip or host name.
localhost will probably pass in most cases

File: test/io/cloudslang/base/network/verify_app_is_up.inputs.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+#
+####################################################
+testVerifyAppIsUp_Success:
+  inputs:
+    - host: 64.233.177.113
+    - port: '80'
+    - attempts: '1'
+  description: Tests that verify_app_is_up.sl operation does not contain an error message
+  testFlowPath: io.cloudslang.base.network.verify_app_is_up
+  result: SUCCESS
```
Comment (oritstone) [inline test/io/cloudslang/base/network/verify_app_is_up.inputs.yaml]: Missing verification of the output_message value

File: test/io/cloudslang/base/network/verify_app_is_up.inputs.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+testVerifyAppIsUp_Failure:
+  inputs:
+    - host: localhost
+    - port: '80'
+    - attempts: '0'
+  description: Tests that verify_app_is_up.sl operation does not contain an error message
+  testFlowPath: io.cloudslang.base.network.verify_app_is_up
+  result: FAILURE
```
Comment (oritstone) [inline test/io/cloudslang/base/network/verify_app_is_up.inputs.yaml]: Missing verification of the output_message value - should contain the error message

**Pull request merged**